docs/contributing/getting_ready.rst

```diff
@@ -228,8 +228,8 @@ environment`_ instead.
    open a terminal, or can be added to the appropriate configuration
    file (i.e., :file:`.bashrc` for bash or :file:`.zshrc` for zsh).
 
-Installing your clone of PlasmaPy
-=================================
+Installing your clone of PlasmaPy 🦹
+===================================
```

Comment:
```suggestion
Installing your clone of PlasmaPy 🦹
====================================
```
...since it appears emojis count as three characters when matching underlines with titles. 🤔